datahub-web-react/src/app/govern/structuredProperties/StructuredPropsTable.tsx

```diff
@@ -55,31 +54,30 @@ const StructuredPropsTable = ({
     setIsViewDrawerOpen,
     selectedProperty,
     setSelectedProperty,
-    inputs,
-    searchAcrossEntities,
+    fetchData,
+    totalCount,
+    setTotalCount,
+    pageSize,
+    searchResults,
+    newProperty,
+    updatedProperty,
 }: Props) => {
     const entityRegistry = useEntityRegistry();
-    const client = useApolloClient();
     const me = useUserContext();
     const canEditProps = me.platformPrivileges?.manageStructuredProperties;
 
-    const structuredProperties = data?.searchAcrossEntities?.searchResults || [];
-
-    // Filter the table data based on the search query
-    const filteredProperties = structuredProperties
-        .filter((prop: any) => prop.entity.definition?.displayName?.toLowerCase().includes(searchQuery.toLowerCase()))
-        .sort(
-            (propA, propB) =>
-                ((propB.entity as StructuredPropertyEntity).definition.created?.time || 0) -
-                ((propA.entity as StructuredPropertyEntity).definition.created?.time || 0),
-        );
```

Comment:
in fact we could continue doing frontend filtering if there is a search query based on the display name! that way we don't have to worry about the weird matching with other fields and only show things that make sense